tic_tac_toe_board.py

```diff
@@ -0,0 +1,47 @@
+class Board():
+    def __init__(self):
+        self.spaces = [None] * 9
+
+    def is_empty_at(self, space):
+        return self.spaces[space] is None
+
+    def mark(self, space, symbol):
+        if self.is_empty_at(space):
+            self.spaces[space] = symbol
+        else:
+            raise ValueError("Invalid Move !")
+
+    def __getitem__(self,space):
+         return self.spaces[space]
+
+    def is_a_way_to_win_a_board(self):
+
+        winningCombos = [[0, 1, 2], [3, 4, 5], [6, 7, 8],
+                        [0, 3, 6], [1, 4, 7], [2, 5, 8],
+                        [0, 4, 8], [6, 4, 2]]
+
+        for group in winningCombos:
+
+            if self.spaces[group[0]] == self.spaces[group[1]] == self.spaces[group[2]]:
+
+                if self.spaces[group[0]] == 'x':
+                    return "X won board"
```

Comment:
Think about how you want to *use* the result of this function. Do you need to be able to make decisions based on it? If so, returning something more useful than a string meant for printing might be prudent.